backends/core/models.py

```diff
@@ -74,6 +91,41 @@ def recipients(self):
       return self.emails_for_notifications.split()
     return []
 
+  def _build_pipeline_prefix(self):
+    return '%s_' % str(self.id)
+
+  def _add_or_update_multi_to_cache(self, mapping, max_retires=10, 
+    time=MEMCACHE_DEFAULT_EXPIRATION_TIME_SECONDS):
```

Comment:
Good call to indent this line, though it would be better with a 4 spaces indentation, to not collide with the implementation :)

Reply:
done